armory/data/adversarial/ucf101_mars_perturbation_and_patch_adversarial_112x112.py

```diff
@@ -249,7 +255,11 @@ def _generate_examples(self, data_dir_path):
                         "adversarial_perturbation": adv_pert,
                         "adversarial_patch": adv_patch,
                     },
-                    "label": label,
+                    "labels": {
+                        "clean": label,
+                        "adversarial_perturbation": label,  # untargeted, label not used
+                        "adversarial_patch": labels[(i + 1) % len(_LABELS)],  # targeted
```

Comment:
Can you document this a little more? You're doing round-robin targeted attacks, specifically?

Reply:
Yes, I can document and update the VERSION.